pyproject.toml

```diff
@@ -65,13 +65,7 @@ protected-access,
 no-self-use,
 arguments-differ,
 attribute-defined-outside-init,
-too-many-statements,
 too-many-instance-attributes,
-too-complex,
-too-many-lines,
-redefined-builtin,
-abstract-method,
-too-many-branches,
```

Comment:
Instead of removing these here and adding them in `tox.ini` for other packages and in files of packages other than `linalg`, should we maybe make a dedicated PR discussing which of these we want to keep in here in the long run?
This way we disentangle PRs and can discuss things with all maintainers

Reply:
- [ ] remove changes to `tox.ini`